client/my-sites/stats/_modernized-mixins.scss

```diff
@@ -89,6 +89,7 @@ $common-border-radius: 4px;
 			font-weight: 500;
 			line-height: 20px;
 			color: var(--studio-black);
+			text-decoration: none;
```

Comment:
This and the above text decoration remove the underline from the links

![Arc -2025-04-04 at 16 19 10@2x](https://github.com/user-attachments/assets/049a0f5e-b07c-4a1b-8bf6-e6e22603e839)

Reply:
Https://github.com/Automattic/wp-calypso/pull/102437 might resolve it.